### DISCUSSION TITLE
Some options are missing on the sidebar ?

### DISCUSSION THREAD

**Discussion by SpidFightFR** [PrismLauncher/PrismLauncher#72] [General]
It looks like some options [are missing from the sidebar,](https://i.imgur.com/9onmTKx.png) is it meant to stay that way or is it temporary ?

I noticed this because of muscle memory, many shortcuts were here to access files and such a bit faster...

**Comment by forkiesassds** [comment]
They were removed

**Comment by TayouVR** [comment]
happened here: https://github.com/PrismLauncher/PrismLauncher/pull/33

we have been meaning to clean up the sidebar for a while, as it was very crowded.
Those buttons could be re-added at some point, but maybe in a different place and better visible/less cluttered.

**Reply by SpidFightFR** [reply]
Oh okay… Just re-adding the "browse .minecraft" would be great, especially on linux where dots files and folder are hidden.

**Reply by theolodger** [reply]
open .minecraft is still present under 'edit' (formerly 'edit instance')

**Reply by TayouVR** [reply]
> Oh okay… Just re-adding the "browse .minecraft" would be great, especially on linux where dots files and folder are hidden.

.folders are always hidden by default, press CTRL+H in your file explorer to show hidden files.

**Reply by SpidFightFR** [reply]
Having a shortcut would be better, too.
